tests/installation/addon_products_sle.pm

```diff
@@ -92,14 +103,9 @@ sub run() {
             if ((split(/,/, get_var('ADDONURL')))[-1] ne $addon) {              # if $addon is not first from all ADDONS
                 send_key 'alt-a';                                               # add another add-on
             }
-            else {
-                send_key 'alt-n';                                               # next
-            }
         }
     }
-    else {
-        send_key 'alt-n', 3;                                                    # done
-    }
+    send_key 'alt-n', 3;                                                        # done
```

Comment:
please replace this 3 by 1 or remove it and use e.g. wait_still_screen
lot of people think it is timeout and it will continue confuse :)